pkgs/by-name/nr/nrc-wrapper-go/package.nix

```
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "nrc-wrapper-go";
+  version = "v1.3.8-2";
```

```suggestion
  version = "1.3.8-2";
```
versions must start with a digit: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning